contracts/hot-cold-aa.sol

```diff
@@ -140,72 +153,88 @@ contract HotColdMultiSig is IAccount, IERC1271 {
         }
     }
 
-    function executeTransactionFromOutside(Transaction calldata _transaction)
-        external 
-        payable 
-        {
-            /*
+    function executeTransactionFromOutside(Transaction calldata _transaction) external payable {
```

Comment:
You can not to implement the logic for this function. Just make `revert("Not implemented")` instead of the logic below. 

The reason is that `executeTransactionFromOutside` is needed for the escape mode, maybe out of scope for this work.